symengine/lib/symengine_wrapper.pyx

```diff
@@ -462,7 +463,6 @@ cdef class Basic(object):
         return Basic._richcmp_(A, B, op)
 
     def _richcmp_(Basic A, Basic B, int op):
-        from sympy import Rel
```

Comment:
For `==` and `!=` it's the same. If I do the same test with `<`, there's a 50%-ish overhead compared to global import.